tests/datasource/fluent/test_spark_filesystem_datasource.py

```diff
@@ -983,6 +987,42 @@ class TimeRange:
             assert metadata[key2] == range2
 
 
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "batch_slice,expected_batch_count",
```

Comment:
nit: can you add a space after the comma.

Reply:
Fixed -- in multiple places that I saw -- thanks.